libs/src/services/hedgehog/index.js

```diff
@@ -63,17 +63,24 @@ class HedgehogWrapper {
      * @param {String} username username
      */
     hedgehog.generateRecoveryInfo = async () => {
-      let entropy = await WalletManager.getEntropyFromLocalStorage()
+      const entropy = await WalletManager.getEntropyFromLocalStorage()
       if (entropy === null) {
         throw new Error('generateRecoveryLink - missing entropy')
       }
-      let btoa = window.btoa
-      if (!btoa) {
-        throw new Error('generateRecoveryLink - missing required btoa function')
+      let convertBinaryToBase64Ascii
+      let currentHost
+      if (typeof window !== 'undefined') {
+        convertBinaryToBase64Ascii = window.btoa
```

Comment:
This just prevents hedgehog in libs from throwing an error when run outside of a browser context. I also changed `btoa` to a more verbose and clear name.